crates/sui-config/Cargo.toml

```diff
@@ -20,17 +20,16 @@ serde_with.workspace = true
 serde_yaml.workspace = true
 tracing.workspace = true
 prometheus.workspace = true
+clap.workspace = true
+object_store.workspace = true
+reqwest.workspace = true
 
 narwhal-config.workspace = true
 sui-keys.workspace = true
 sui-protocol-config.workspace = true
-sui-storage.workspace = true
 sui-types.workspace = true
```

Comment:
By the way, sui-types still depend on rocksdb.
So while this is a good cleanup, it may not achieve what you want in the end.

Reply:
hm how are you checking it? i tried importing sui-sdk and sui-types to a separating rust repo and didn't see rocksdb

Reply:
Ah you are right. Looks like that was already removed previously.